docs/walk-through.md

```diff
@@ -40,215 +42,231 @@ For each required certificate you will need to know its *thumbprint*. Both conde
+Open `token.txt` in your favorite text editor; you'll see that the secured token is substantially longer than an unsecured one.
 
-The server will start up and wait for connections.
+### Storing the token in a variable
+
+Set the environment variable `AZ_BATCH_SOFTWARE_ENTITLEMENT_TOKEN` to the content of the `token.txt` file:
 
+``` PowerShell
+# PowerShell
+PS> $env:AZ_BATCH_SOFTWARE_ENTITLEMENT_TOKEN = (get-content tmp\token.txt)
 ```
-17:20:02.676 [Information] ---------------------------------------------
-17:20:02.695 [Information]   Software Entitlement Service Test Utility
-17:20:02.696 [Information] ---------------------------------------------
-17:20:02.977 [Debug] Hosting starting
-17:20:03.043 [Debug] Hosting started
-Hosting environment: Production
-Content root path: ... elided ...
-Now listening on: https://localhost:4443
-Application started. Press Ctrl+C to shut down.
+``` bash
+# bash
+$ AZ_BATCH_SOFTWARE_ENTITLEMENT_TOKEN=`cat tmp/token.txt`
 ```
 
-Test the server with a web browser, connecting to the server by entering the URL shown on the console.
-
-![Browser](../img/browser.png)
-
-Only an HTTPS connection will work. The server does not listen for HTTP connections. If you specified any kind of locally signed certificate, you will likely need to override security features of your browser to connect.
-
-Leave the server running in this shell window and return to the first. You may want to arrange the two shell windows so you can observe the one running the test server while you work in the original window.
+Testing with an encrypted token is useful because these are significantly longer, in part due to information about the required key that's included within. (This may reveal issues, such as buffer sizes that are too small, that an unencrypted token would not.) Note that your application doesn't need to do anything different with encrypted vs unencrypted tokens. 
 
 ## Verifying a token
 
-In your original shell window, define an environment variable with the URL shown by the server:
+There are a few options for independently checking the software entitlement token. This allows you to check the consistency of the system without involving your application, helping you to identify the source of any issues.
+
+In the same shell window used to generate the token, define an environment variable with the URL shown by the server:
 
 ``` PowerShell
-$env:AZ_BATCH_ACCOUNT_URL = "https://localhost:4443"
+# PowerShell
+PS> $env:AZ_BATCH_ACCOUNT_URL = "https://localhost:4443"
+```
+``` bash
+# bash
+$ AZ_BATCH_ACCOUNT_URL="https://localhost:4443"
 ```
 
-To independently check a software entitlement token, use the `sesclient` console application. This allows you to check the consistency of the system without involving your application, helping you to identify the source of any issues. 
+You should now have the following variables defined:
+- `AZ_BATCH_SOFTWARE_ENTITLEMENT_TOKEN`
+- `AZ_BATCH_ACCOUNT_URL`
+- `applicationId`
```

Comment:
Do we need to call out that the first two are environment variables and the last one is a shell variable?